io/net/network_test.py

```diff
@@ -204,7 +216,7 @@ def receive_offload_toggle_test(self, ro_type, ro_type_full):
 
     def receive_offload_state_change(self, ro_type, ro_type_full, state):
```

Comment:
@manvanthar Just a suggestion on cosmetic change. Considering this function is going to change TSO as well now, can we name this `offload_state_change()` ?